Intersect.Server/Entities/Npc.cs

```diff
@@ -32,6 +32,24 @@ public class Npc : Entity
         //Damage Map - Keep track of who is doing the most damage to this npc and focus accordingly
         public ConcurrentDictionary<Entity, long> DamageMap = new ConcurrentDictionary<Entity, long>();
 
+        // Returns the entity that ranks the highest on this NPC's damage map.
+        public Entity DamageMapHighest { 
+            get {
+                var damageMap = DamageMap;
```

Comment:
This is just a direct reference to `DamageMap` but it's a concurrent collection anyway, just use it directly in the for-each.